python/ecl/summary/ecl_sum_vector.py

```diff
@@ -99,8 +100,20 @@ def days(self):
     def mpl_dates(self):
         """
         All the dates as numpy vector of dates in matplotlib format.
+        This property will be replaced by numpy_dates, but is kept for
+        backwards-compatibility for the time-being. Usage will trigger
+        a depreciation warning.
         """
-        return self.__mpl_dates
+        warning.warn("The mpl_dates property has been deprecated - use numpy_dates instead",
+                     DeprecationWarning)
+        return self.__numpy_dates
```

Comment:
You are now changing the behaviour of the function you are deprecating. Should continue to return `self.__mpl_dates` until removed.

Reply:
Yes, I see. I did this to avoid having to init __mpl_dates. See next comment; that might be a better solution.